Makefile

```diff
@@ -1,5 +1,5 @@
 PD_PKG := github.com/pingcap/pd
-VENDOR := $(shell rm -f _vendor/src/$(PD_PKG) &&\
+VENDOR := $(shell rm -rf _vendor/src/$(PD_PKG) &&\
```

Comment:
is it better to set GOPATH with vendor here directly, so we don't need to link the vendor path anymore?